### PR TITLE
Add option not to launch db in baxter_moveit_config

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

File: baxter/baxter_moveit_config/launch/demo_dummy.launch [hunk 1]
```diff
@@ -38,5 +38,12 @@
     <arg name="rviz_config" value="$(arg rviz_config)" />
   </include>
 
-  <include file="$(find baxter_moveit_config)/launch/default_warehouse_db.launch" />
+  <!-- If this package is installed, setting 'db' as 'true' or db_path to user permission dir is required. -->
```
Comment (v4hn) [inline baxter/baxter_moveit_config/launch/demo_dummy.launch]: don't you mean "setting 'db' as '***false***' or db_path ..."?

Comment (wkentaro) [inline baxter/baxter_moveit_config/launch/demo_dummy.launch]: You're right.

File: baxter/baxter_moveit_config/launch/default_warehouse_db.launch [hunk 2]
```diff
@@ -3,8 +3,9 @@
   <arg name="reset" default="false"/>
 
   <!-- Launch the warehouse with a default database location -->  
+  <arg name="db_path" default="$(find baxter_moveit_config)/default_warehouse_mongo_db" />
```
Comment (v4hn) [inline baxter/baxter_moveit_config/launch/default_warehouse_db.launch]: This argument is already present in [moveit_setup_assistent's template](https://github.com/ros-planning/moveit_setup_assistant/blob/indigo-devel/templates/moveit_config_pkg_template/launch/default_warehouse_db.launch).
Could you please adapt the change to mirror the template?
I suppose, it's not a good idea to diverge too far from the template.

**Comment by v4hn** [comment]
+1
@wkentaro the next dev who looks at this will probably merge it.

**Pull request merged**